src/components/Section/Section.js

```diff
@@ -3,14 +3,15 @@ import PropTypes from "prop-types";
 
 import "./Section.css";
 
-const Section = ({ bgColor, children }) => {
+const Section = ({ customClass, bgColor, children }) => {
   return (
-    <section className={`Section Section--bg-${bgColor}`}>{children}</section>
+    <section className={`Section Section--bg-${bgColor} ${customClass}`}>{children}</section>
   );
 };
 
 Section.propTypes = {
   bgColor: PropTypes.node,
```

Comment:
Are You sure that this is node (html element) ? Maybe it's string. If label could be node write something like this
label: PropTypes.oneOfType([ PropTypes.string, PropTypes.node ]),